madmom/audio/cepstrogram.py

```diff
@@ -222,6 +245,63 @@ def __new__(cls, spectrogram, filterbank=MelFilterbank,
         # return the object
         return obj
 
+    @staticmethod
+    def calc_deltas(data, delta_filter):
+        """
+        Applies the given filter to the data after automatically padding by
+        replicating the first and last frame. The length of the padding is
+        calculated via ceil(len(delta_filter))
+        :param data: a numpy array with the data
+        :param delta_filter: a numpy array containing the filter
+        :return: a numpy array of the same shape as data, containing the deltas
```

Comment:
Please use [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) style docstrings. Also, try following [PEP257](https://www.python.org/dev/peps/pep-0257/) (e.g. "Apply the...").